src/test/kotlin/sirius/db/jdbc/SmartQueryKotlinTest.kt

```diff
@@ -31,24 +31,24 @@ class SmartQueryKotlinTest {
     @Test
     fun `queryList returns all entities`() {
         val smartQueryTestEntity = oma.select(SmartQueryTestEntity::class.java)
-                .orderAsc(SmartQueryTestEntity.TEST_NUMBER)
+            .orderAsc(SmartQueryTestEntity.TEST_NUMBER)
```

Comment:
Looks as if we need to optimise our Kotlin formatting rules 😇

Reply:
They are (or rather should be when the IDE is configured correctly) defined by the Kotlin language themselves. It is highly discouraged to stray away from it and create a customized formatting ruleset.
But this formatting looks like the language default isn't applied here.
Maybe we can change something in sirius-parent to make usage easier.